codes/quantum/qubits/rotated_concatenated_stabilizer.yml

```diff
@@ -0,0 +1,36 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: rotated_concatenated_stabilizer
+physical: qubits
+logical: qubits
+
+name: 'Rotated Concatenated Stabilizer Codes'
+introduced: '\cite{arXiv:2010.00538}'
+
+description: |
+
+  A \([[2n,k,d]]\) rotated concatenated stabilizer code is a quantum error-correcting code constructed by concacenating an outer \([[n,k,d]]\) stabilizer code with the inner dual-rail code which encodes \(|\bar{0}\rangle=|01\rangle\) and \(|\bar{1}\rangle=|10\rangle\)\cite{arXiv:2010.00538}.
+
+protection: |
+
+  A rotated concatenated stabilizer code is a constant-excitation code, hence immune to coherent phase errors acting identically and simultaneously on all qubits\cite{arXiv:2010.00538}.
+
+
+  The rotated concatenated stabilizer code obtained by choosing the outer stabilizer code to be the 4-qubit code devised in \cite{arXiv:quant-ph/9704002} can correct a single amplitude damping error\cite{arXiv:2010.00538}.
+
+features:
+
+  transversal_gates: 'If the outer stabilizer code has a transversal encoded operation realizable by a tensor product of single-qubit Paulis, then the same encoded operation can also be implementedd as a tensor product of single-qubit Paulis in the concatenated code\cite{arXiv:2010.00538}.'
+
+  threshold:
+    - 'If the outer stabilizer code is a CSS code with transversal logical single-qubit Paulis given by \(\overline{P}=P^{\otimes n}\) for every \(P\in\{I,X,Y,Z\}\), then the concatenated code has a positive fault-tolerant threshold w.r.t stochastic noise\cite{arXiv:2010.00538}.'
+
+relations:
+  parents:
+    - code_id: quantum_concatenated
+
+  cousins:
```

Comment:
cousin: stab_4_2_2 with detail about relevant subcodes (LNCY and ABC+ in paper)
cousin: dual_rail with detail about how it is concatenated with a stab code